docs/zh_CN/Installation.md

```diff
@@ -1,8 +1,8 @@
 # 安装 NNI
 
-当前仅支持 Linux 和 Mac。
+当前支持 Linux, Mac 和 Windows。
 
-## **安装**
+## **在 Linux 和 Mac 上安装**
```

Comment:
Same as above. Please do not modify the Chinese document directly here. NNI unifies the Chinese corresponding document in another place.